app/auth/templates/register.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Register</title>
+</head>
+<body>
+    <h1>Register</h1>
+    <form action="/register" method="POST">
+        <label for="username">Username:</label><br>
+        <input type="text" id="username" name="username"><br>
+        <label for="email">Email:</label><br>
+        <input type="email" id="email" name="email"><br>
+        <label for="password">Password:</label><br>
+        <input type="password" id="password" name="password"><br>
+        <button type="submit">Register</button>
+    </form>
+</body>
+</html>
```

Comment:
Looks good! We would want the app to be modular though and `auth` to contain only auth code. Would you please include html code related to authentication in a directory under templates.
**Eg.**
`app/templates/auth/register.html`

**NB**: Look also `login.html` code.